### PR TITLE
change style

### PR DESCRIPTION
**Pull request by MoisesFlores22**
What this PR does?
Changes the style of the chat to be more according to the mocks for twg

Any background context you want to provide?

🚧
The webchat needed the correct style for the delivering of the twg bot

How can this be manually tested?
Run the chat and text everything is according to mocks

- [x] @AlexisGlez 

- [x] @JoseILopez 

- [ ] @rguinea 

- [ ] @ricardogrw 

- [x] @castrosalazr

### CODE REVIEW

File: src/Shell.tsx [hunk 1]
```diff
@@ -36,40 +44,82 @@ class ShellContainer extends React.Component<Props, {}> {
         this.sendMessage();
     }
 
-    private onChangeFile() {
-        this.textInput.focus();
-        this.props.sendFiles(this.fileInput.files, this.props.user, this.props.format.locale);
-        this.fileInput.value = null;
+    private openMenu = () => {
+        console.log('eeee');
+        console.log(this.state)
```
Comment (JoseILopez) [inline src/Shell.tsx]: Remove logging please

File: src/Shell.tsx [hunk 2]
```diff
@@ -36,40 +44,82 @@ class ShellContainer extends React.Component<Props, {}> {
...
+    }
+
+    private Troubleshooting = () =>{
+        this.props.sendMessage('Troubleshooting', this.props.user, this.props.format.locale);
+        this.closeMenu();
+    }
+
+    private specsProduct = () =>{
+        this.props.sendMessage('Product Specs', this.props.user, this.props.format.locale);
+        this.closeMenu();
```
Comment (JoseILopez) [inline src/Shell.tsx]: You could probably just use a single function for the three menu buttons. Just pass the parameter to use in sendMessage. Kind of like this:

`private menuMessage = (message) => {
this.props.sendMessage(message, this.props.user, this.props.format.locale);
this.closeMenu();
}`

File: src/Shell.tsx [hunk 3]
```diff
@@ -36,40 +44,82 @@ class ShellContainer extends React.Component<Props, {}> {
...
+                    <img className={"ic-menu"} src={imgSrc} />
+                    </label>
+                    <div className="wc-textbox">
+                        <input
+                            type="text"
+                            ref={ input => this.textInput = input }
+                            autoFocus
+                            value={ this.props.inputText }
+                            onChange={ _ => this.props.onChangeText(this.textInput.value) }
+                            onKeyPress={ e => this.onKeyPress(e) }
```
Comment (AlexisGlez) [inline src/Shell.tsx]: Here you could just use:
`onKeyPress={ this.onKeyPress }`
The param is going to be passed to your this.onKeyPress function.

Comment (AlexisGlez) [inline src/Shell.tsx]: And instead of e it'd be better if you use event

File: src/Shell.tsx [hunk 4]
```diff
@@ -36,40 +44,82 @@ class ShellContainer extends React.Component<Props, {}> {
...
+                            placeholder={ this.props.format.strings.consolePlaceholder }
+                        />
```
Comment (AlexisGlez) [inline src/Shell.tsx]: You could place this at the end of the line 116:
`placeholder={ this.props.format.strings.consolePlaceholder } />`

File: src/Shell.tsx [hunk 5]
```diff
@@ -36,40 +44,82 @@ class ShellContainer extends React.Component<Props, {}> {
...
+                    </div>
+                    <label className="wc-send" onClick={ () => this.onClickSend() } >
+                        SEND
```
Comment (AlexisGlez) [inline src/Shell.tsx]: Same as above, you could use the uppercase className, but its up to you :)

File: wz-botchat.css [hunk 6]
```diff
@@ -478,9 +508,16 @@ ul.wc-card-buttons li {
...
-    background-color: #000;
-    color: #fff !important;
-    border-radius: 4px;
+    color: #4d4d4d !important;
+    border-radius: 8px;
+    display: flex;
+}
+
+.bot::before {
+    content: url(/static/assets/logo.png);
```
Comment (AlexisGlez) [inline wz-botchat.css]: I don't know if you can do this here, remember when @jfer-rubio said something about this that could crash in production?

File: wz-botchat.css [hunk 7]
```diff
@@ -539,11 +583,20 @@ ul.wc-card-buttons li {
 }
 
 .wc-send {
-    right: 0;
+  font-size: 16px;
+  font-style: normal;
+  font-stretch: normal;
+  color: #cccccc;
+  padding: 15px 10px;
+  right: 0;
```
Comment (AlexisGlez) [inline wz-botchat.css]: This entire className has less spaces at the beginning than the others.

File: wz-botchat.css [hunk 8]
```diff
@@ -539,11 +583,20 @@ ul.wc-card-buttons li {
...
 }
 
-.wc-console.has-text .wc-send svg {
-    fill: #3A96DD;
+.ic-menu {
+    padding: 18px 10px;
+}
+
+.wc-console.has-text .wc-send {
+  color: #0083ff;
```
Comment (AlexisGlez) [inline wz-botchat.css]: This entire className has less spaces at the beginning than the others.

File: wz-botchat.css [hunk 9]
```diff
@@ -583,10 +636,90 @@ ul.wc-card-buttons li {
...
+    margin: 0;
+    padding: 0;
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    right: 0;
+    left: 0;
+    overflow: hidden;
+	background-color: rgba(0, 0, 0, 0.5);
+	border: solid 1px #979797;
```
Comment (AlexisGlez) [inline wz-botchat.css]: Line 649 and 650 have more initial spaces.

File: wz-botchat.css [hunk 10]
```diff
@@ -583,10 +636,90 @@ ul.wc-card-buttons li {
...
+    color: #0084ff !important;
+    background-color: #ffffff !important;
+    width: 100%;
+    height: 44px;
+    bottom: 86px;
+    display: flex;
+    box-shadow: inset -2px 0 0 0 #ededed, inset 2px 0 0 0 #ededed, inset 0 -2px 0 0 #ededed;
+}
+
+.Specs-button {
```
Comment (AlexisGlez) [inline wz-botchat.css]: Here is the uppercased classname jeje

File: src/Menu.tsx [hunk 11]
```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
```
Comment (MoisesFlores22) [inline src/Menu.tsx]: is necesary other way it just don't work 😢

File: src/History.tsx [hunk 12]
```diff
@@ -218,8 +218,8 @@ export class WrappedActivity extends React.Component<WrappedActivityProps, {}> {
             default:
                 let sent: string;
                 if (this.props.showTimestamp)
-                    sent = this.props.format.strings.timeSent.replace('%1', (new Date(this.props.activity.timestamp)).toLocaleTimeString());
-                timeLine = <span>{ this.props.activity.from.name || this.props.activity.from.id }{ sent }</span>;
+                    sent = "sent " + this.props.format.strings.timeSent.replace('%1', (new Date(this.props.activity.timestamp)).toLocaleTimeString());
```
Comment (MoisesFlores22) [inline src/History.tsx]: that solution makes that i could not write anything in the input 🤔

File: wz-botchat.css (hunk 6 above)
Comment (MoisesFlores22) [inline wz-botchat.css]: the problem here is that i need that element and i can't place it any other way 🤔

File: src/History.tsx (hunk 12 above)
Comment (AlexisGlez) [inline src/History.tsx]: 🤔  what happened when the code entered to the if statement?

File: wz-botchat.css (hunk 6 above)
Comment (AlexisGlez) [inline wz-botchat.css]: Ok, then :)

File: src/History.tsx (hunk 12 above)
Comment (MoisesFlores22) [inline src/History.tsx]: microsoft magic :trollface:, i don't know is something i didn't touch XD just change the `sent` label

Comment (MoisesFlores22) [inline src/History.tsx]: when you press a key it change the internal timestamp

Comment (AlexisGlez) [inline src/History.tsx]: ok try this one:
`const sent = this.props.showTimestamp ? `sent ${replacedDate}` : undefined;`

Comment (MoisesFlores22) [inline src/History.tsx]: nop

Comment (AlexisGlez) [inline src/History.tsx]: Ok try this:
```const sent = this.props.showTimestamp ? `sent ${this.props.format.strings.timeSent.replace('%1', (new Date(this.props.activity.timestamp)).toLocaleTimeString())}` : ‘’;```

File: samples/backchannel/index.html [hunk 13]
```diff
@@ -41,7 +41,7 @@
 </head>
 
 <body>
-	<h2 style="font-family:Segoe UI;">Type a color into the WebChat!</h2>
+	<h2 style="font-family:Nunito;">Type a color into the WebChat!</h2>
```
Comment (castrosalazr) [inline samples/backchannel/index.html]: 😛 I'm not a big fan of inline CSS, why not create a class or something.

File: wz-botchat.css [hunk 14]
```diff
@@ -282,16 +287,38 @@ body .wc-app, .wc-app button, .wc-app input, .wc-app textarea {
 
 ul.wc-card-buttons {
     clear: both;
-    display: table;
+    display: flex;
+    flex-direction: column;
+}
+
+ul.wc-card-buttons li:hover {
+    clear: both;
```
Comment (castrosalazr) [inline wz-botchat.css]: Is this `clear: both` necessary? Was it overridden? Also, if I'm not mistaken, I think `clear` and `float` are ignored when using flexbox.

File: wz-botchat.css [hunk 15]
```diff
@@ -454,9 +483,10 @@ ul.wc-card-buttons li {
 
 /* from me */
 .me {
-    background-color: #000 !important;
-    color: #fff;
-    border-radius: 4px;
+    background-color:  #1d2c5f !important;
```
Comment (castrosalazr) [inline wz-botchat.css]: 😔 try using  `!important` as less as possible; maybe by overriding the precedence of your CSS rules, i.e. `.my-container .me`.

File: wz-botchat.css [hunk 16]
```diff
@@ -583,10 +636,90 @@ ul.wc-card-buttons li {
...
+    padding: 0 20px 0 20px !important;
+    color: #0084ff !important;
+    background-color: #ffffff !important;
+    width: 100%;
+    height: 44px;
+    bottom: 43px;
+    display: flex;
+    box-shadow: inset -2px 0 0 0 #ededed, inset 2px 0 0 0 #ededed, inset 0 -2px 0 0 #ededed;
+}
+.user-button {
```
Comment (castrosalazr) [inline wz-botchat.css]: #ocd Missed an enter

**Pull request merged**